src/sutta_processor/application/domain_models/bilara_translation/root.py

```diff
@@ -65,10 +65,10 @@ def from_file_paths(cls, exclude_dirs: List[Path], file_paths: List[Path]) -> "B
     @classmethod
     def _update_index(cls, index: dict, file_aggregate):
         def get_lang() -> str:
-            for part in reversed(file_aggregate.f_pth.parts):
-                if part in {"en", "de", "jpn", "pt", "ru"}:
+            for part in file_aggregate.f_pth.parts:
```

Comment:
Reversing didn't help anything.  It actually caused a few extra unnecessary iterations since the language is towards the front of the path.